opm/simulators/wells/BlackoilWellModelGeneric.cpp

```diff
@@ -279,7 +279,7 @@ initFromRestartFile(const RestartValue& restartValues,
     }
 
 
-    this->active_wgstate_.well_test_state = std::move(wtestState);
+    this->active_wgstate_.wtest_state(wtestState);
```

Comment:
Is there no benefit to using `std::move()` here?  In other words, do you not get any improvements if you say
```C++
this->active_wgstate_.wtest_state(std::move(wtestState));
```
in this context?